src/index.html

```diff
@@ -9,6 +9,22 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="1200">
+      <a href="#" scr="link"><img src="images/logo.png" alt="logo"></a>
+      <nav>
+        <ul>
+          <li><a href="#" class="is-active"></a>Apple</a></li>
+          <li><a href="#" scr="link"></a>Samsung</a></li>
+          <li><a href="#" scr="link"></a>Smartphones</a></li>
+          <li><a href="#" data-qa="hover"></a>Laptops & Computers</a></li>
+          <li><a href="#" scr="link"></a>Gadgets</a></li>
+          <li><a href="#" scr="link"></a>Tablets</a></li>
+          <li><a href="#" scr="link"></a>Photo</a></li>
+          <li><a href="#" scr="link">Video</a></li>
```

Comment:
```suggestion
          <li>
            <a href="#" class="is-active">Apple</a>
          </li>
          <li>
            <a href="#" scr="link">Samsung</a>
          </li>
          <li>
            <a href="#" scr="link">Smartphones</a>
          </li>
          <li>
            <a href="#" data-qa="hover">Laptops & Computers</a>
          </li>
          <li>
            <a href="#" scr="link">Gadgets</a>
          </li>
          <li>
            <a href="#" scr="link">Tablets</a>
          </li>
          <li>
            <a href="#" scr="link"></a>Photo</a>
          </li>
          <li>
            <a href="#" scr="link">Video</a>
          </li>
```
Consider fixing indents according to the code style guide